src/ms-store-badge.d.ts

```diff
@@ -13,6 +13,18 @@ declare class MSStoreBadge extends HTMLElement {
      * The ID of your app.
      */
     productId: string;
+    /**
+     * The optional campaign ID of your app.
+     */
+    cid: string;
+    /**
+      * Indicates whether popup or full mode should be launched.
+      */
+    popup: string;
```

Comment:
Is this really a string? If the value is either popup or full, we should type it like this:

`popup: "popup" | "full"`

Also, is popup the right name? It doesn't convey meaning. If you haven't already, Hong or Daniel might have suggestions.